intelligent-video-recording/README.md

```diff
@@ -4,22 +4,22 @@
 
 The demo aims to be a reference intelligent video recording security camera. The configuration file (''config.cfg'') allows to add a list of classes ID from the model being used.
 
-As seen on the following image when a defined class ID is detected it will start recording. Default uses class ID 762 (Remote Control). If the class ID is still being detected it will continue recording, if the class ID does not match it will wait ``MIN_RECORDING_TIME_IN_SECONDS`` before saving the video. Multiple classes ID can be detected at once. If after saving the video a new valid detection appears a new video file will be created.
+As seen on the following image when a defined class ID is detected it will start recording. Default uses class ``Remote Control``. If the class ID is still being detected it will continue recording, if the class ID does not match it will wait ``MIN_RECORDING_TIME_IN_SECONDS`` before saving the video. Multiple classes ID can be detected at once. If after saving the video a new valid detection appears a new video file will be created.
```

Comment:
Changes: 

As seen on the following image when a defined class ID is detected``,`` it will start recording. The default uses class ``Remote Control``. If the class ID is still being detected`,` it will continue recording, if the class ID does not match it will wait ``MIN_RECORDING_TIME_IN_SECONDS`` before saving the video. Multiple classes ID can be detected at once. If after saving the video a new valid detection appears`,` a new video file will be created.

Please, check this redaction, it is not clear this phrase 

`Default uses class ``Remote Control`,  What is default? 
`if the class ID does not match`. Regarding to?